validator/app/src/compute_horde_validator/validator/synthetic_jobs/v1_decrypt.py

```diff
@@ -11,10 +11,15 @@
 
 
 def run_cmd(cmd):
-    try:
-        return subprocess.check_output(cmd, shell=True).decode()
-    except Exception:
-        return ""
+    proc = subprocess.run(cmd, shell=True, capture_output=True, check=False)
+    stdout = proc.stdout.decode()
+    if proc.returncode != 0:
+        stderr = proc.stderr.decode()
+        raise RuntimeError(
+            f"[errno {proc.returncode}] {cmd}\nSTDOUT:\n{stdout}\nSTDERR:\n{stderr}"
+        )
+    return stdout
```

Comment:
suggestion:
a) text=True takes care of decoding for you
b) repr escapes output properly
```suggestion
    proc = subprocess.run(cmd, shell=True, capture_output=True, check=False, text=True)
    if proc.returncode != 0:
        raise RuntimeError(
            f"run_cmd error {cmd=!r} {proc.returncode=} {proc.stdout=!r} {proc.stderr=!r}"
        )
    return proc.stdout
```